src/wallet/wallet_rpc_server.cpp

```diff
@@ -4759,6 +4759,22 @@ namespace tools
     return true;
   }
   //------------------------------------------------------------------------------------------------------------------------------
+  bool wallet_rpc_server::on_get_default_fee_priority(const wallet_rpc::COMMAND_RPC_GET_DEFAULT_FEE_PRIORITY::request& req, wallet_rpc::COMMAND_RPC_GET_DEFAULT_FEE_PRIORITY::response& res, epee::json_rpc::error& er, const connection_context *ctx)
+  {
+    if (!m_wallet) return not_open(er);
+    try
+    {
+      res.priority = m_wallet->adjust_priority(0) - 1;
```

Comment:
Most code (including the function called here) reserves zero priority to mean "don't care"/"default", and distinguishes between the zero-based *index* of a priority in some array from the actual priority. So, if this field is called priority, it should probably return the one-based value, which you can map to an index on the client-side. Or, rename the field to priority_index, which I think would make its meaning equally clear.

Reply:
Ok I changed it to use one-based priority.